### DISCUSSION TITLE
Mobile view not working on ios for uploaded GLB files.

### DISCUSSION THREAD

**Discussion by ishook187** [google/model-viewer#4364] [Q&A]
I tried on an iPhone and iPad with the QR code connection. One test file was just a sphere with a blue texture. Files would load onto the preview area of the phone, but wouldn't do anything when the view in AR was clicked. The high-res astronaut and many of the other files managed to load into AR, although some would get stuck on the loading bar on the iphone. I'd really like to get my own GLB loading in AR successfully .

**Comment by elalish** [comment]
Is this a dupe of #4271? Please add repro details there if so.

**Comment by ishook187** [comment]
Yes - Sorry I didn't see there was already one this similar. I'll comment on there. Thank you!
